data_enrichment.ipynb

```diff
@@ -0,0 +1,505 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Missing Tabular Data Enrichment With YOU Research API \n",
+    "\n",
+    "In this tutorial, we will enrich the missing values in a tabular dataset primarily using `You.com` research API. \n",
+    "\n",
+    "Firstly, set the environment variable `YDC_API_KEY` with your `You.com` api key."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "os.environ[\"YDC_API_KEY\"] = \"<YOUR YOU.COM API KEY>\""
```

Comment:
Nit: I know the old examples do it like this, but for the new ones we are suggesting people using dotenv to load creds from a .env file. Checkout the latest examples for how to do it like this.

Reply:
For example: https://github.com/You-OpenSource/You-API-Examples/blob/main/legal_assistant_csv_langchain_chatbot.ipynb